src/components/team-member-add/TeamMemberAdd.tsx

```diff
@@ -158,11 +167,11 @@ const TeamMemberAdd = ({ addFormData, handleFormVisible }) => {
             <input
               className={`${styles.input} 
               ${validity.homeAdress === false ? styles.invalidInput : ''}`}
-              maxLength="30"
-              minLength="2"
+              maxLength={30}
+              minLength={2}
               name="homeAdress"
               onBlur={handleBlur}
-              onChange={({ target }) => setHomeAdress(target.value)}
+              onChange={({ target }): void => setHomeAdress(target.value)}
```

Comment:
Factually you're returning the return value of the `setHomeAddress` function. Just remember that because of this it's not necessarily `void`.